src/Sulu/Bundle/MediaBundle/DataFixtures/ORM/LoadCollectionTypes.php

```diff
@@ -24,43 +24,28 @@ class LoadCollectionTypes extends AbstractFixture implements OrderedFixtureInter
      */
     public function load(ObjectManager $manager)
     {
-        // force id = 1
+        // set id manually
         $metadata = $manager->getClassMetaData(CollectionType::class);
         $metadata->setIdGeneratorType(\Doctrine\ORM\Mapping\ClassMetadata::GENERATOR_TYPE_NONE);
 
-        $defaultCollectionType = $this->createCollectionType(1, 'collection.default', 'Default');
-        $manager->persist($defaultCollectionType);
+        // create or update collectiontype with id 1
+        $defaultCollectionType = new CollectionType();
+        $defaultCollectionType->setId(1);
+        $defaultCollectionType = $manager->merge($defaultCollectionType);
```

Comment:
I would move the merge call after all the setters of this entity have been called.